src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java

```diff
@@ -44,10 +49,20 @@ private void process() {
 				stdProcessor.process(dataProcessId);
 			}
 
+			break;
+		case 2:
+			System.out.println("How many records you want to insert ? : ");
+			int recordCount1 = scanner.nextInt();
```

Comment:
why it is recordCount1 instread recordCount ?

Reply:
Yes sir i changed